### DISCUSSION TITLE
about the copy assignment feature

### DISCUSSION THREAD

**Discussion by kayasax** [kayasax/EasyPIM#75] [Ideas]
so I'm starting to work on this.

I added a -assignee parameter to get-pimazureresourceeligible, this give a list of all eligible assignement for a specific user.
Then you wan to create new eligible assignements for the same roles to someone else right? We will use the maximum eligible duration for the new assignments?
Smthg like this?

![image](https://github.com/user-attachments/assets/97878dd5-6bc3-4a85-baad-7b239d1c52cf)

**Comment by Azure365Addict** [comment]
Hi @kayasax,

Sorry for a late reply. 

Yes, I was looking for something like this 👍

Would be great if it would be possible to do the same for Entra Roles 🤔

**Reply by kayasax** [reply]
Hey @Azure365Addict  hope you are enjoying the feature.
I'm a bit reluctant to do the same for Entra roles....
Initialy you said the problem was with group synced from onprem. For entra roles the good practice is to use cloud-only users (like your on prem admin should not be synced) ....

**Comment by Azure365Addict** [comment]
Hi @kayasax,

I agree that using cloud-only users is a good practice, but it seems to work best in greenfield environments.

For admin accounts (synced from on-prem), in addition to using PIM, we also utilize a third-party PAM tool to ensure just-in-time access for enhanced security.

**Comment by kayasax** [comment]
@Azure365Addict added the Copy-PIMEntraRoleEligibleAssignment commandlet to V1.7.7
